source/MissionAction.cpp

```diff
@@ -186,11 +194,13 @@ void MissionAction::Load(const DataNode &node, const string &missionName)
 			conversation.Load(child);
 		else if(key == "conversation" && hasValue)
 			stockConversation = GameData::Conversations().Get(child.Token(1));
+		else if(key ==  "ship" && hasValue)
+			giftShips[GameData::Ships().Get(child.Token(1))] = (child.Size() > 2 ? child.Token(2) : "");
```

Comment:
Should we print a warning when trying to give a ship that doesn't exist in the GameData?

Reply:
Don't we already get an "(outfit | ship) X is referred to but never defined" warning?

Reply:
Good point! Yes that is probably handled by a generic check on empty ship definitions after loading.

Reply:
So I don't need to worry about trying to add a checks here? Nice!